aten/src/ATen/native/mps/operations/ConstantOps.mm

```diff
@@ -79,16 +63,26 @@
 }
 
 Tensor& zero_mps_(Tensor& self) {
-  return at::native::fill_scalar_mps_impl(self, 0.0f);
+  if (self.is_contiguous()) {
```

Comment:
Do we need to extend this if-check for other cases ? Did we miss any thing from view side ?

Reply:
As we tested, no extra conditions are required.